<launch>
  <node pkg="move_base" type="move_base" name="move_base" output="screen">
    <rosparam file="$(find sr_navigation)/param/costmap_common_params.yaml" command="load" ns="global_costmap"/>
    <rosparam file="$(find sr_navigation)/param/costmap_common_params.yaml" command="load" ns="local_costmap"/>
    <rosparam file="$(find sr_navigation)/param/global_costmap_params.yaml" command="load"/>
    <rosparam file="$(find sr_navigation)/param/dwa_local_planner_params.yaml" command="load"/>
    <rosparam file="$(find sr_navigation)/param/local_costmap_params.yaml" command="load"/>
    <rosparam file="$(find sr_navigation)/param/global_planner_params.yaml" command="load"/>
    <rosparam file="$(find sr_navigation)/param/move_base_params.yaml" command="load"/>
  </node>
</launch>
